restclient/src/main/java/com/box/l10n/mojito/rest/resttemplate/FormLoginAuthenticationCsrfTokenInterceptor.java

```diff
@@ -46,9 +47,6 @@ public class FormLoginAuthenticationCsrfTokenInterceptor implements ClientHttpRe
 
   @Autowired FormLoginConfig formLoginConfig;
 
-  /** The {@link AuthenticatedRestTemplate} to which this interceptor is being used. */
-  @Autowired AuthenticatedRestTemplate authRestTemplate;
```

Comment:
Removing AuthenticatedRestTemplate from FormLoginAuthenticationCsrfTokenInterceptor.java fixes the circular dependency issue